pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -80,7 +80,15 @@
     abstract class BaseCommand extends CliCommand {
         @Override
         void run() throws Exception {
-            processArguments();
+            try {
+                processArguments();
+            } catch (Exception e) {
+                System.err.println(e.getMessage());
```

Comment:
Can we use Logger.error instead of Syserr.

Reply:
I think we should use Syserr here, no. It prints the error message before showing the jcommand helper. so people knows what is the error.